buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -73,7 +73,7 @@ object Dependencies {
     const val slf4j_api = "org.slf4j:slf4j-api:2.0.0-alpha0"
     const val slf4j_timber = "com.arcao:slf4j-timber:3.1@aar"
     const val emoji_java = "com.vdurmont:emoji-java:5.1.1"
-    const val json_schema_validator = "com.github.java-json-tools:json-schema-validator:2.2.14"
+    const val json_schema_validator = "com.networknt:json-schema-validator:1.0.66"
```

Comment:
This framework is less "starred" on Github but actually seems to be more active and brings in a far less heavy set of dependencies (its README mentions this being intentional as well).